src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Added support for Activity Status and StatusDescription which were
+  added to Activity from `System.Diagnostics.DiagnosticSource` version 6.0. To maintain backward
+  compatibility, the exporter falls back to checking status inside
+  the tag "otel.status_code".
+ ([#3100](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3100))
```

Comment:
I've always thought of our use of `otel.status_code` as an internal implementation detail. I wouldn't want someone to read this and think they should set these tags manually when using a version of  `System.Diagnostics.DiagnosticSource` prior to 6.0, so I took a stab at elaborating a little more. What do you think?

```suggestion
* Added support for Activity Status and StatusDescription which were
  added to Activity from `System.Diagnostics.DiagnosticSource` version 6.0.
  Prior to version 6.0, setting the status of an Activity was provided by the
  .NET OpenTelemetry API via the `Activity.SetStatus` extension method in the
  `OpenTelemetry.Trace` namespace. Internally, this extension method added the
  status as tags on the Activity: `otel.status_code` and `otel.status_description`.
  Therefore, to maintain backward compatibility, the exporter falls back to using
  these tags to infer status.
 ([#3100](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3100))
```